Bottom: f5d66eb10e9cf6e99e038c41afeec88f72f751b2
Top:    79b36b9ed22f77cfe81e888155e383bcf09e5e9f
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-06-05 16:22:01 -0700

Refresh of rs-spin

---

diff --git a/src/rsocket.c b/src/rsocket.c
index 3619af4..bb66503 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -66,7 +66,7 @@
 static struct index_map idm;
 static pthread_mutex_t mut = PTHREAD_MUTEX_INITIALIZER;
 
-static unsigned long long polling_time;
+static uint32_t polling_time;
 
 /*
  * Immediate data format is determined by the upper bits
@@ -206,7 +206,7 @@ void rs_configure(void)
 	FILE *f;
 
 	if ((f = fopen(RS_CONF_DIR "/polling_time", "r"))) {
-		fscanf(f, "%Lu", &polling_time);
+		fscanf(f, "%u", &polling_time);
 		fclose(f);
 	}
 }
@@ -941,7 +941,7 @@ static int rs_process_cq(struct rsocket *rs, int nonblock, int (*test)(struct rs
 static int rs_get_comp(struct rsocket *rs, int nonblock, int (*test)(struct rsocket *rs))
 {
 	struct timeval s, e;
-	unsigned long long poll_time = 0;
+	uint32_t poll_time = 0;
 	int ret;
 
 	do {
@@ -1508,7 +1508,7 @@ int rpoll(struct pollfd *fds, nfds_t nfds, int timeout)
 {
 	struct timeval s, e;
 	struct pollfd *rfds;
-	unsigned long long poll_time = 0;
+	uint32_t poll_time = 0;
 	int ret;
 
 	do {
